app/application/interfaces/services.py

```
@@ -53,37 +67,36 @@ async def create_events_bulk(
         trigger_workflows: bool = False,
     ) -> list[EventEntity]:
         """Bulk create events (e.g. email sync)."""
-        ...
 
 
+# Workflow engine interface
 class IWorkflowEngine(Protocol):
     """Protocol for workflow execution triggered by events."""
 
     async def process_event_triggers(self, event: Any, tenant_id: str) -> list[Any]:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `event`

(ANN401)